material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -668,10 +667,19 @@ input.md-search__input::placeholder {
     padding-left: 1em;
   }
 
-  [dir="ltr"] .md-nav--primary .md-nav__item > .md-nav__link {
+  [dir='ltr'] .md-nav--primary .md-nav__item > .md-nav__link {
     margin-right: 0;
   }
 
+  .md-sidebar--primary
+    .md-nav__item
+    .md-nav
+    > .md-nav__list
+    > .md-nav__item
+    > a.md-nav__link {
+    padding-right: 1.5em;
```

Comment:
This CSS selector seems too specific. We have to make sure this is being applied to all the nav items. Even in the TOC, otherwise the same thing could happen.

For example this is a TOC item I hovered over and added to the title:

<img width="292" height="64" alt="Image" src="https://github.com/user-attachments/assets/9861c7d8-6c1e-4249-9f13-7ac3184fec1a" />